app/components/page_list_component/_index.scss

```diff
@@ -0,0 +1,3 @@
+.govuk-button-group-right {
```

Comment:
Is this overriding govuk design system styling?

Should we not perhaps add our own "app" specific override?

Reply:
yes good point! I've updated the CSS, based off the same convention we used in the task-list.